weblate/trans/migrations/0164_alter_component_push_alter_component_repo.py

```diff
@@ -27,9 +27,9 @@ class Migration(migrations.Migration):
             model_name="component",
             name="repo",
             field=models.CharField(
-                help_text="URL of a repository, use weblate://project/component to share it with other component.",
+                help_text="The URL of a repository. Use weblate://project/component to use the settings of an existing Weblate component.",
```

Comment:
Sharing is a better description of what happens there – the repository is cloned just once in Weblate.